src/auth/jwtaccess.ts

```diff
@@ -150,7 +149,7 @@ export class JWTAccess {
     callback?: (err?: Error) => void
   ): void | Promise<void> {
     if (callback) {
-      this.fromStreamAsync(inputStream).then(r => callback(), callback);
+      this.fromStreamAsync(inputStream).then(() => callback(), callback);
```

Comment:
Just to be cool, would ".then(callback, callback)" work, too?

Reply:
Theoretically yes.  For readability, I like to expand this out for folks.

Reply:
Probably the trickier thing here (for non-Promise users and non-JS users [who would also be non-Promise users]), is the "then(successHandler, errorHandler)" convention. If we're not all-in on the shortcut by using `callback` consistently, I think we'd be most readable these ways:

- `.then(callback).catch(callback)`
- `.then(() => callback()).catch(err => callback(err))`
- `.then(() => callback(), err => callback(err))`
- `.then(callback, callback)`

Reply:
That is entirely fair 😆 I have some weird historical aversion to just passing the function reference, because it kicked me in the face once.  Chances are I'm being weird, and you're right that I should just pass the function reference (example #4).